invenio_records_global_search/services/services.py

```diff
@@ -42,3 +48,55 @@ def create_or_update(
         except PIDDoesNotExistError:
             self.record_cls.gs_pid = gs_pid
             return self._create(self.record_cls, identity, data, uow=uow, expand=expand)
+
+    def search(
+        self,
+        identity: Identity,
+        params: dict | None = None,
```

Comment:
is there really a case where `params` can be `None` except the default value? so makes it sense to use `search` without `params`?
the same applies for `search_preference`.
in the super() class they have no default value